File: img/prediction/state_of_art/sst1.c
void dm_x_bcs(int rank) {
    ...
    mpi->sendrecv(ny+2, sstmac::sw::mpytype::mpi_real,\
    proc_x_max, tag, ny+2, sstmac::sw::mpitype::mpi_real,\
    proc_x_min, tag, world(), stat);
    ...
}

File: img/prediction/state_of_art/sst1.f
SUBROUTINE dm_x_bcs
    ...
    CALL MPI_SENDRECV(dm(nx-1, 0:ny+1), ny+2, mpireal, &
        proc_x_max, tag,dm(-1, 0:ny+1), ny+2, mpireal, &
        proc_x_min, tag, comm, status, errcode)
    ...
END SUBROUTINE dm_x_bcs
